src/components/board.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export const Board = props => {
+    return (
+        <div className="root">
+            <div className="grid-container">{props.lights}</div>
+            <p>{props.numMoves}</p>
+            <button onClick={props.reset}>Reset</button>
+            {props.winComponent}
+        </div>
+    );
+};
+
+Board.propTypes = {
+    lights: React.PropTypes.array,
+    numMoves: React.PropTypes.number,
+    reset: React.PropTypes.func,
+    winComponent: React.PropTypes.element,
```

Comment:
Hmm, I'm getting this error when running locally:

![image](https://user-images.githubusercontent.com/72114/85577865-d9454800-b5f6-11ea-9801-63bf23ced895.png)

I'm _pretty_ sure that this is because `PropTypes` is no longer a part of React as of about 2 years ago — now, you need to install + import the `prop-types` library separately and use it instead. `React.PropTypes` totally used to work, which is why you can still find blog posts that use this format, but I don't believe it'll work anymore.